malaria-app-android/src/main/java/com/peacecorps/malaria/db/DatabaseSQLiteHelper.java

```diff
@@ -393,11 +393,14 @@ public int getDosesInaRowDaily()
                 } catch (Exception e) {
                     return 0;
                 }
-                if (cursor.getString(0).compareTo("yes") == 0) {
-                    prevDate = cursor.getInt(2);
-                    prevDateMonth = cursor.getInt(3);
-                    if (Math.abs(currDate - prevDate) <= 1)
-                        dosesInaRow++;
+
+                if(cursor.getString(0)!=null){
+                    if (cursor.getString(0).compareTo("yes") == 0){
+                        prevDate = cursor.getInt(2);
+                        prevDateMonth = cursor.getInt(3);
+                        if (Math.abs(currDate - prevDate) <= 1)
+                            dosesInaRow++;
+                }
                 }

```

Comment:
The brackets should have a different level of closing as they are of different order.